books/free-programming-books-langs.md

```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### Bash
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://www.pdfdrive.com/bash-cookbook-solutions-and-examples-for-bash-users-e184776614.html) - Carl Albing & JP Vossen (PDF)
```

Comment:
remove this

Reply:
@CyberSecByte

Reply:
Ok I'm gonna close this and make a new pull request